<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabTools_InfoLookup</class>
 <widget class="QWidget" name="TabTools_InfoLookup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TabTools_InfoLookup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>langSelect.label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="langSelectComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>songSelect</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="SongIdSelector" name="songIdSelector" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>packInfo</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1" columnminimumwidth="0,100">
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>pack.description</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="packIdLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>pack.id</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="packDescriptionLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>pack.name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="packNameLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="songInfoGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>songInfo</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1" columnminimumwidth="0,150">
         <item row="6" column="0">
          <widget class="QLabel" name="label_23">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.bgDayNight</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_27">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.audioPreview</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="songAudioPreviewLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="songArtistLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.artist</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="songSourceLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="songTitleLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_19">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.bgSide</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.id</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.date&amp;version</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="songBgSideLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="songIdLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="songDateVersionLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="songBgDayNightLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.title</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.bpm</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_25">
           <property name="styleSheet">
            <string notr="true">QLabel { color: gray; }</string>
           </property>
           <property name="text">
            <string>song.source</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="songBpmLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>chartSelect</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="RatingClassSelector" name="ratingClassSelector" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>chartInfo</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1" columnminimumwidth="0,50">
           <item row="0" column="1">
            <widget class="QLabel" name="chartConstantLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="chartNotesLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>chart.constant</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>chart.notes</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>difficultyInfo</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0,0,1" columnminimumwidth="0,50,0,0,50">
           <item row="0" column="3">
            <widget class="QLabel" name="label_18">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.date&amp;version</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="difficultyRatingLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QLabel" name="difficultyJacketNightLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="difficultyBgLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="difficultyBgInverseLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_31">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.artist</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.rating</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_33">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.bg</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="difficultyChartDesignerLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="difficultyAudioOverrideLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="difficultyBpmLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLabel" name="label_43">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.jacketOverride</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_22">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.chartDesigner</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QLabel" name="difficultyJacketOverrideLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_35">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.audioOverride</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_34">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.bpm</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_41">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.bgInverse</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="difficultyDateVersionLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_26">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.jacketDesigner</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLabel" name="label_42">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.jacketNight</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_29">
             <property name="styleSheet">
              <string notr="true">QLabel { color: gray; }</string>
             </property>
             <property name="text">
              <string>difficulty.title</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="difficultyArtistLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="difficultyJacketDesignerLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="difficultyTitleLabel">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="6">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SongIdSelector</class>
   <extends>QWidget</extends>
   <header>ui.implements.components.songIdSelector</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RatingClassSelector</class>
   <extends>QWidget</extends>
   <header>ui.implements.components.ratingClassSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
